Automattic/wp-calypso
# Devdocs: WordPress components gallery

#### Changes proposed in this Pull Request

* Add a new WordPress components gallery page to the devdocs

<img width="1456" alt="Captura de Tela 2020-09-11 às 10 32 25" src="https://user-images.githubusercontent.com/24264157/92955499-1a8d0d00-f41a-11ea-8334-3ab96be1d767.png">

#### Testing instructions



* Using an environment that has devdocs enabled, navigate to the new WordPress components gallery and check that each component is represented.

Note that some components are represented through the building of the page itself, for example, `Card` et co as well as the `Flex` module were used to build the page, so I didn't feel it was necessary to include them as distinct examples. Actually using them is probably the best way to show that they work.
